### DISCUSSION TITLE
Tutorial manual 2023

### DISCUSSION THREAD

**Discussion by vjagodnik** [Anura3D/Anura3D_OpenSource#115] [Q&A] [answered]
Hello to everyone,
I am having an issue with TutorialManual_2023 and 2022.
No matter how I tried to download it, the file has 134 bytes.
With kind regards
V

**Comment by ayerrocolom** [comment]
If you download the full code from [https://github.com/Anura3D/Anura3D_OpenSource](https://github.com/Anura3D/Anura3D_OpenSource) (select "Code" and "Download ZIP"), you will find the tutorial manual in the zip file under "manuals".

**Reply by vjagodnik** [reply]
I did and experiencing the same issue:
![Screenshot 2023-07-04 at 13 03 02](https://github.com/Anura3D/Anura3D_OpenSource/assets/36809874/177c18d2-dbd4-44ae-9c44-fab55ae7dd19)

![Screenshot 2023-07-04 at 13 03 46](https://github.com/Anura3D/Anura3D_OpenSource/assets/36809874/5a67bf08-2567-48e4-8e2f-362b91820768)

**Reply by ayerrocolom** [reply]
Hello! We are working on solving this issue. The actual pdf is too big. We apologise for the inconvenience. We will let you know as soon as this is solved.

**Reply by ayerrocolom** [reply, marked as answer]
Hello,
Can you please try again in [https://github.com/Anura3D/Anura3D_OpenSource/blob/main/manuals/TutorialManual_2023.pdf](https://github.com/Anura3D/Anura3D_OpenSource/blob/main/manuals/TutorialManual_2023.pdf)? If you cannot visualize it on your browser, click on "Raw" to download the file.

**Reply by vjagodnik** [reply]
Works fine now 
LFS rules 😃 
Thank you
